server/api/util.go

```diff
@@ -21,9 +21,24 @@ import (
 	"net/http"
 
 	"github.com/juju/errors"
+	"github.com/pingcap/pd/pkg/apiutil"
 	"github.com/pingcap/pd/server"
+	"github.com/unrolled/render"
 )
 
+func jsonRespondError(r *render.Render, w http.ResponseWriter, body io.ReadCloser, data interface{}) (err error) {
```

Comment:
The function name _jsonRespondError_ conceals the fact that it reads json from `body`. Maybe something like _readJSONRespondError_?